File: coj.uci.cu/1417 - Drop/Main.java
/*  Problem: Drop
 *  Code: 1417 - Drop
 *  Site: coj.uci.cu
 *  Status: pending
 *  Type: WA
 */

import java.io.*;
import java.text.*;

public class Main
{
	
	public static void main (String args[]) throws IOException
	{
		InputStreamReader isr= new InputStreamReader(System.in);
		BufferedReader in=new BufferedReader(isr);		
		double n;
		int j;
		boolean flag=false;			
		while(true)
		{
			n=Double.parseDouble(in.readLine());			
			if(n==0.0)
			  break;
			if(flag)
			   System.out.println();
			j=1;
			flag=true;            
			System.out.printf("Starting height: %.5f meter(s)\n",n);
			while(n>=1.0/1000.0)
			{
				n=n/2.0;
				
				System.out.printf("Bounce #%d: %.10f meters\n",j,n);
				j++;
			}
		}
	}
}

File: coj.uci.cu/1417 - Drop/main.cxx
/*  Problem: Drop
 *  Code: 1417 - Drop
 *  Site: coj.uci.cu
 *  Status: pending
 *  Type: WA
 */



#include <iostream>
#include <cstdio>
using namespace std;
int main(int argc, char **argv)
{
	long double n;
	int j;
	bool flag=false;
	while(1)
	{
		cin>>n;
		if(n==0.0)
		   break;
		if(flag)
		   cout<<endl;
		flag=true;
		printf("Starting height: %.5Lf meter(s)\n",n);
		j=1;
		while(n>=(1.0/1000.0))
		{
			n=n/2.0;
			printf("Bounce #%d: %.10Lf meters\n",j,n);
			j++;
		}
	}
	
	return 0;
}

